### ISSUE TITLE
Valheim server restarts intermittently

### ISSUE THREAD

**Issue by EnigmaRaptor** [lloesche/valheim-server-docker#180]
I can't find the pattern, but the server will restart from time to time without warning.  I'm monitoring my Linux resources and I don't see any anomalies in the RAM, CPU, or network usage at the time the server restarts.  I don't know where to look for more details, is there a log file I can view?

**Comment by InB4DevOps** [comment]
docker logs container-name

**Comment by lloesche** [comment]
@EnigmaRaptor did you find more info? Like @InB4DevOps said `docker logs <container-name>` will give you the complete logs. They are pretty extensive so whatever is going wrong we should see in there.

**Comment by altrae** [comment]
Not sure if this is the same as my issue, but for me it is not random and is happening on a schedule. Check your logs as suggested because mine restarts at 5am UTC every day no matter if players are connected or not. Just posted my issue in Q&A because I didn't find another thread prior:
https://github.com/lloesche/valheim-server-docker/discussions/187

**Comment by EnigmaRaptor** [comment]
@lloesche unfortunately I have not had time to work on this yet. I have changed my online ENV var to 1 on some advice from other server operators in the valheim discord channel, but no time to review logs yet. I will report back my findings.

**Comment by EnigmaRaptor** [comment]
I have reviewed the logs and it looks as though the logs are persistent only through the existing run, so the logs leading up to the crash are not available.  We experienced one crash this night, which is fewer than normal.  I can provide the logs for the current run, but I don't know what good that will do.

**Comment by EnigmaRaptor** [comment]
2021-03-07 23:01:51,308 WARN No file matches via include "/etc/supervisor/conf.d/*.conf"
,2021-03-07 23:01:51,308 INFO Set uid to user 0 succeeded
,2021-03-07 23:01:51,332 INFO RPC interface 'supervisor' initialized
,2021-03-07 23:01:51,333 INFO supervisord started with pid 1
,2021-03-07 23:01:52,335 INFO spawned: 'valheim-bootstrap' with pid 8
,2021-03-07 23:01:52 [8] (DEBUG) Creating cron to do world backups using schedule 0 * * * *
,2021-03-07 23:01:52,364 INFO success: valheim-bootstrap entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
,2021-03-07 23:01:52 [8] (DEBUG) Creating cron to check for updates using schedule */15 * * * *
,2021-03-07 23:01:52 [8] (DEBUG) Creating cron to restart valheim-server using schedule 0 5 * * *
,2021-03-07 23:01:52,504 INFO spawned: 'syslogd' with pid 20
,Mar  7 23:01:52 c2c76113de28 syslog.info syslogd started: BusyBox v1.32.1
,2021-03-07 23:01:53,509 INFO success: syslogd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
,syslogd: started
,2021-03-07 23:01:53,637 INFO spawned: 'crond' with pid 22
,Mar  7 23:01:53 c2c76113de28 cron.info crond[22]: crond (busybox 1.32.1) started, log level 8
,2021-03-07 23:01:54,640 INFO success: crond entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
,crond: started
,2021-03-07 23:01:54,760 INFO spawned: 'valheim-updater' with pid 24
,2021-03-07 23:01:54 [24] (DEBUG) Running Valheim Server updater as user root uid 0
,2021-03-07 23:01:54 [24] (DEBUG) Valheim updater was just started - skipping connected players check
,2021-03-07 23:01:54 [24] (INFO) Downloading/updating/validating Valheim server from Steam
,2021-03-07 23:01:54 [24] (DEBUG) Found CPU with 3315.979 MHz
,2021-03-07 23:01:55,944 INFO success: valheim-updater entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
,valheim-updater: started
,2021-03-07 23:01:56,066 INFO spawned: 'valheim-backup' with pid 58
,2021-03-07 23:01:56 [58] (INFO) Backing up Valheim server worlds to /config/backups/worlds-20210307-230156.zip
,  adding: worlds/ (stored 0%)
,  adding: worlds/Dedicated.fwl.old (deflated 4%)
,  adding: worlds/Dedicated.fwl (deflated 4%)
,  adding: worlds/Dedicated.db.old2021-03-07 23:01:57,172 INFO success: valheim-backup entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
,valheim-backup: started
,2021-03-07 23:01:57,188 INFO exited: valheim-bootstrap (exit status 0; expected)
, (deflated 69%)
,  adding: worlds/Dedicated.db (deflated 69%)
,2021-03-07 23:02:00 [58] (INFO) Removing backups older than 3 days
,/config/backups/worlds-20210305-040004.zip
,Redirecting stderr to '/root/Steam/logs/stderr.txt'
,[  0%] Checking for available updates...
,[----] Verifying installation...
,[  0%] Downloading update...
,[  0%] Checking for available updates...
,[----] Download complete.
,[----] Extracting package...
,[----] Extracting package...
,[----] Extracting package...
,[----] Extracting package...
,[----] Installing update...
,[----] Installing update...
,[----] Installing update...
,[----] Installing update...
,[----] Installing update...
,[----] Installing update...
,[----] Installing update...
,[----] Installing update...
,[----] Cleaning up...
,[----] Update complete, launching Steamcmd...
,Redirecting stderr to '/root/Steam/logs/stderr.txt'
,[  0%] Checking for available updates...
,[----] Verifying installation...
,Steam Console Client (c) Valve Corporation
,-- type 'quit' to exit --
,Loading Steam API...OK.
,
,Connecting anonymously to Steam Public...Logged in OK
,Waiting for user info...OK
, Update state (0x3) reconfiguring, progress: 0.00 (0 / 0)
, Update state (0x3) reconfiguring, progress: 0.00 (0 / 0)
, Update state (0x5) verifying install, progress: 8.43 (88683656 / 1051599543)
, Update state (0x5) verifying install, progress: 36.66 (385522664 / 1051599543)
, Update state (0x5) verifying install, progress: 64.16 (674754744 / 1051599543)
, Update state (0x5) verifying install, progress: 89.64 (942645501 / 1051599543)
,Success! App '896660' fully installed.
,.d..t...... ./
,2021-03-07 23:02:27 [24] (INFO) Valheim Server is already the latest version
,2021-03-07 23:02:27 [24] (DEBUG) Writing file to start Valheim server
,2021-03-07 23:02:27,679 INFO spawned: 'valheim-server' with pid 122
,2021-03-07 23:02:27 [122] (INFO) Running Valheim Server
,2021-03-07 23:02:27 [122] (DEBUG) Waiting for server to listen on port 2456
,Found path: /opt/valheim/server/valheim_server.x86_64
,Mono path[0] = '/opt/valheim/server/valheim_server_Data/Managed'
,Mono config path = '/opt/valheim/server/valheim_server_Data/MonoBleedingEdge/etc'
,Preloaded 'libsteam_api.so'
,Initialize engine version: 2019.4.20f1 (6dd1c08eedfa)
,[Subsystems] Discovering subsystems at path /opt/valheim/server/valheim_server_Data/UnitySubsystems
,Forcing GfxDevice: Null
,GfxDevice: creating device client; threaded=0
,NullGfxDevice:
,    Version:  NULL 1.0 [1.0]
,    Renderer: Null Device
,    Vendor:   Unity Technologies
,Begin MonoManager ReloadAssembly
,- Completed reload, in  0.072 seconds
,UnloadTime: 1.824098 ms
,03/07/2021 23:02:28: Starting to load scene:start
,2021-03-07 23:02:32 [122] (DEBUG) Waiting for server to listen on port 2456
,2021-03-07 23:02:37 [122] (DEBUG) Waiting for server to listen on port 2456
,2021-03-07 23:02:37,713 INFO success: valheim-server entered RUNNING state, process has stayed up for > than 10 seconds (startsecs)
,valheim-server: started
,HDR Render Texture not supported, disabling HDR on reflection probe. 
,HDR Render Texture not supported, disabling HDR on reflection probe. 
,Unloading 6 Unused Serialized files (Serialized files now loaded: 0)
,UnloadTime: 4.310552 ms
,Only custom filters can be played. Please add a custom filter or an audioclip to the audiosource (Amb_MainMenu). 
,HDR Render Texture not supported, disabling HDR on reflection probe. 
,03/07/2021 23:02:38: Initializing world generator seed: ( 0 )   menu:True  worldgen version:1
,03/07/2021 23:02:38: Using mountain distance: 1000
,03/07/2021 23:02:38: Render threading mode:SingleThreaded
,03/07/2021 23:02:38: Get create world Dedicated
,03/07/2021 23:02:38: Using environment steamid 892970
,03/07/2021 23:02:38: Using steam APPID:892970
,[S_API] SteamAPI_Init(): Loaded local 'steamclient.so' OK.
,CAppInfoCacheReadFromDiskThread took 1 milliseconds to initialize
,CApplicationManagerPopulateThread took 0 milliseconds to initialize (will have waited on CAppInfoCacheReadFromDiskThread)
,RecordSteamInterfaceCreation (PID 126): SteamGameServer013 / 
,RecordSteamInterfaceCreation (PID 126): SteamUtils009 / 
,Setting breakpad minidump AppID = 892970
,RecordSteamInterfaceCreation (PID 126): SteamGameServer013 / GameServer
,RecordSteamInterfaceCreation (PID 126): SteamUtils009 / Utils
,RecordSteamInterfaceCreation (PID 126): SteamNetworking006 / Networking
,RecordSteamInterfaceCreation (PID 126): SteamGameServerStats001 / GameServerStats
,RecordSteamInterfaceCreation (PID 126): STEAMHTTP_INTERFACE_VERSION003 / HTTP
,RecordSteamInterfaceCreation (PID 126): STEAMINVENTORY_INTERFACE_V003 / Inventory
,RecordSteamInterfaceCreation (PID 126): STEAMUGC_INTERFACE_VERSION014 / UGC
,RecordSteamInterfaceCreation (PID 126): STEAMAPPS_INTERFACE_VERSION008 / Apps
,[S_API FAIL] Tried to access Steam interface SteamNetworkingUtils003 before SteamAPI_Init succeeded.
,RecordSteamInterfaceCreation (PID 126): SteamNetworkingUtils003 / 
,RecordSteamInterfaceCreation (PID 126): SteamNetworkingSockets008 / 
,03/07/2021 23:02:38: Server ID 90071992547409920
,Warning: failed to set thread priority: set failed for 8: -1: setpriority() failed
,Warning: failed to set thread priority: set failed for priority 8
,Warning: support for elevated priorities is most likely unavailable, suppressing future warnings
,03/07/2021 23:02:38: Authentication:k_ESteamNetworkingAvailability_Waiting
,03/07/2021 23:02:38: Steam game server initialized
,03/07/2021 23:02:38: Missing audio clip in music respawn
,03/07/2021 23:02:38: Builder started
,03/07/2021 23:02:39: Loaded localization English
,03/07/2021 23:02:39: Game server connected
,Unloading 16 unused Assets to reduce memory usage. Loaded Objects now: 85988.
,Total: 60.330455 ms (FindLiveObjects: 3.580741 ms CreateObjectMapping: 9.983291 ms MarkObjects: 46.464146 ms  DeleteObjects: 0.301306 ms)
,Unloading 6 Unused Serialized files (Serialized files now loaded: 0)
,UnloadTime: 11.686972 ms
,HDR Render Texture not supported, disabling HDR on reflection probe. 
,HDR Render Texture not supported, disabling HDR on reflection probe. 
,03/07/2021 23:02:40: Zonesystem Awake 1059
,03/07/2021 23:02:40: DungeonDB Awake 1059
,03/07/2021 23:02:40: Registering lobby
,03/07/2021 23:02:40: Initializing world generator seed:yQvAa7Vxgx ( 2077906916 )   menu:False  worldgen version:1
,03/07/2021 23:02:40: Using mountain distance: 1000
,03/07/2021 23:02:40: Found 465 mountain points
,03/07/2021 23:02:40: Remaining mountains:68
,03/07/2021 23:02:40: Calc time 18.04 ms
,03/07/2021 23:02:40: Found 8122 lake points
,03/07/2021 23:02:42: Remaining lakes:105
,03/07/2021 23:02:42: Calc time 1883.329 ms
,03/07/2021 23:02:42: Rivers:118
,03/07/2021 23:02:42: River buckets 12663
,03/07/2021 23:02:42: River render time 234.085 ms
,03/07/2021 23:02:42: River Calc time 243.74 ms
,03/07/2021 23:02:43: River buckets 21452
,03/07/2021 23:02:43: River render time 435.044 ms
,03/07/2021 23:02:43: Placed 1862 streams
,03/07/2021 23:02:43: Stream Calc time 972.777 ms
,03/07/2021 23:02:43: Load world Dedicated
,03/07/2021 23:02:43: Loading 384391 zdos , my id 193929259 data version:26
,03/07/2021 23:02:44: Loaded 100000 dead zdos
,03/07/2021 23:02:44: Removed 0 OLD generated ZDOS
,03/07/2021 23:02:44: Loaded 7128 locations
,03/07/2021 23:02:44: Audioman already exist, destroying self
,Unloading 508 unused Assets to reduce memory usage. Loaded Objects now: 86321.
,Total: 362.796949 ms (FindLiveObjects: 3.900862 ms CreateObjectMapping: 10.905416 ms MarkObjects: 342.087066 ms  DeleteObjects: 5.902821 ms)
,03/07/2021 23:02:48: Zonesystem Start 1059
,03/07/2021 23:02:48: DungeonDB Start 1059
,03/07/2021 23:02:49: Got status changed msg k_ESteamNetworkingConnectionState_Connecting
,03/07/2021 23:02:49: New connection
,03/07/2021 23:02:49: Accepting connection k_EResultInvalidState
,03/07/2021 23:02:49: Got status changed msg k_ESteamNetworkingConnectionState_ClosedByPeer
,03/07/2021 23:02:49: Socket closed by peer Steamworks.SteamNetConnectionStatusChangedCallback_t
,03/07/2021 23:03:03: Got status changed msg k_ESteamNetworkingConnectionState_Connecting
,03/07/2021 23:03:03: New connection
,03/07/2021 23:03:03: Accepting connection k_EResultOK
,03/07/2021 23:03:03: Connecting to Steamworks.SteamNetworkingIdentity
,03/07/2021 23:03:03: Got status changed msg k_ESteamNetworkingConnectionState_Connected
,03/07/2021 23:03:03: Connected
,03/07/2021 23:03:03: Got connection SteamID 76561198059085861
,03/07/2021 23:03:03: Got handshake from client 76561198059085861
,03/07/2021 23:03:05: Got status changed msg k_ESteamNetworkingConnectionState_Connecting
,03/07/2021 23:03:05: New connection
,03/07/2021 23:03:05: Accepting connection k_EResultOK
,03/07/2021 23:03:05: Connecting to Steamworks.SteamNetworkingIdentity
,03/07/2021 23:03:05: Got status changed msg k_ESteamNetworkingConnectionState_Connected
,03/07/2021 23:03:05: Connected
,03/07/2021 23:03:05: Got connection SteamID 76561197963260490
,03/07/2021 23:03:05: Got handshake from client 76561197963260490
,03/07/2021 23:03:06: VERSION check their:0.147.3  mine:0.147.3
,03/07/2021 23:03:06: Server: New peer connected,sending global keys
,03/07/2021 23:03:08: VERSION check their:0.147.3  mine:0.147.3
,03/07/2021 23:03:08: Server: New peer connected,sending global keys
,03/07/2021 23:03:16: Got status changed msg k_ESteamNetworkingConnectionState_Connecting
,03/07/2021 23:03:16: New connection
,03/07/2021 23:03:16: Accepting connection k_EResultOK
,03/07/2021 23:03:16: Connecting to Steamworks.SteamNetworkingIdentity
,03/07/2021 23:03:16: Got status changed msg k_ESteamNetworkingConnectionState_Connected
,03/07/2021 23:03:16: Connected
,03/07/2021 23:03:16: Got connection SteamID 76561197978303732
,03/07/2021 23:03:16: Got handshake from client 76561197978303732
,03/07/2021 23:03:22: VERSION check their:0.147.3  mine:0.147.3
,03/07/2021 23:03:22: Server: New peer connected,sending global keys
,03/07/2021 23:03:24: Got character ZDOID from Bacon : 115916012:1
,03/07/2021 23:03:26: Got character ZDOID from Hofflander : 1998091234:4
,03/07/2021 23:03:43: Got character ZDOID from Lothar : 2380463480:1
,03/07/2021 23:10:03: Got character ZDOID from Bacon : 0:0
,03/07/2021 23:10:11: Got character ZDOID from Bacon : 115916012:661
,03/07/2021 23:12:49:  Connections 3 ZDOS:384427  sent:404 recv:722
,Unloading 6 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 6 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 620.215266 ms (FindLiveObjects: 65.810820 ms CreateObjectMapping: 20.606145 ms MarkObjects: 532.963150 ms  DeleteObjects: 0.834255 ms)
,Mar  7 23:15:00 c2c76113de28 cron.info crond[22]: USER root pid 542 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-07 23:15:04 [24] (DEBUG) Received signal to check for update
,2021-03-07 23:15:06 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/07/2021 23:15:49: Time 290061.419849988, day:160    nextm:290070.000010729  skipspeed:0.715013395063579
,03/07/2021 23:22:49:  Connections 3 ZDOS:384468  sent:502 recv:410
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 614.807320 ms (FindLiveObjects: 66.417425 ms CreateObjectMapping: 21.044898 ms MarkObjects: 526.566119 ms  DeleteObjects: 0.777672 ms)
,03/07/2021 23:22:50: clone 612
,03/07/2021 23:22:52: Saved 384410 zdos
,03/07/2021 23:22:52: World saved ( 1469.884ms )
,Mar  7 23:30:00 c2c76113de28 cron.info crond[22]: USER root pid 980 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-07 23:30:06 [24] (DEBUG) Received signal to check for update
,2021-03-07 23:30:07 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/07/2021 23:32:50:  Connections 3 ZDOS:384462  sent:390 recv:253
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 610.519451 ms (FindLiveObjects: 69.031030 ms CreateObjectMapping: 20.517557 ms MarkObjects: 520.155766 ms  DeleteObjects: 0.813822 ms)
,03/07/2021 23:34:06: Got status changed msg k_ESteamNetworkingConnectionState_Connecting
,03/07/2021 23:34:06: New connection
,03/07/2021 23:34:06: Accepting connection k_EResultOK
,03/07/2021 23:34:06: Connecting to Steamworks.SteamNetworkingIdentity
,03/07/2021 23:34:06: Got status changed msg k_ESteamNetworkingConnectionState_Connected
,03/07/2021 23:34:06: Connected
,03/07/2021 23:34:06: Got connection SteamID 76561197993893753
,03/07/2021 23:34:06: Got handshake from client 76561197993893753
,03/07/2021 23:34:08: VERSION check their:0.147.3  mine:0.147.3
,03/07/2021 23:34:08: Server: New peer connected,sending global keys
,03/07/2021 23:34:25: Got character ZDOID from Gaelen : 121322511:1
,03/07/2021 23:42:50:  Connections 4 ZDOS:384446  sent:851 recv:552
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 615.911870 ms (FindLiveObjects: 67.830192 ms CreateObjectMapping: 19.903615 ms MarkObjects: 527.467966 ms  DeleteObjects: 0.708955 ms)
,03/07/2021 23:42:51: clone 611
,03/07/2021 23:42:53: Saved 384382 zdos
,03/07/2021 23:42:53: World saved ( 1520.999ms )
,Mar  7 23:45:00 c2c76113de28 cron.info crond[22]: USER root pid 1420 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-07 23:45:08 [24] (DEBUG) Received signal to check for update
,2021-03-07 23:45:09 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/07/2021 23:52:51:  Connections 4 ZDOS:384391  sent:910 recv:586
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 663.646848 ms (FindLiveObjects: 67.644140 ms CreateObjectMapping: 20.815720 ms MarkObjects: 574.349316 ms  DeleteObjects: 0.836294 ms)
,03/07/2021 23:57:25: RPC_Disconnect 
,03/07/2021 23:57:25: Destroying abandoned non persistent zdo 2380463480:1 owner 2380463480
,03/07/2021 23:57:25: Disposing socket
,03/07/2021 23:57:25: Closing socket 76561197978303732
,03/07/2021 23:57:25:   send queue size:0
,03/07/2021 23:57:25: Disposing socket
,03/07/2021 23:57:25: Got status changed msg k_ESteamNetworkingConnectionState_ClosedByPeer
,03/07/2021 23:57:25: Socket closed by peer Steamworks.SteamNetConnectionStatusChangedCallback_t
,03/07/2021 23:57:25: Got status changed msg k_ESteamNetworkingConnectionState_None
,Mar  8 00:00:00 c2c76113de28 cron.info crond[22]: USER root pid 1855 cmd [ -f "/var/run/valheim-backup.pid" ] && kill -HUP $(cat /var/run/valheim-backup.pid)
,Mar  8 00:00:00 c2c76113de28 cron.info crond[22]: USER root pid 1856 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 00:00:00 [24] (DEBUG) Received signal to check for update
,2021-03-08 00:00:01 [58] (DEBUG) Received signal to backup world
,2021-03-08 00:00:01 [58] (INFO) Backing up Valheim server worlds to /config/backups/worlds-20210308-000001.zip
,  adding: worlds/ (stored 0%)
,  adding: worlds/Dedicated.fwl.old (deflated 4%)
,  adding: worlds/Dedicated.fwl (deflated 4%)
,  adding: worlds/Dedicated.db.old2021-03-08 00:00:02 [24] (DEBUG) Players connected to Valheim server - skipping update check
, (deflated 69%)
,  adding: worlds/Dedicated.db (deflated 69%)
,2021-03-08 00:00:04 [58] (INFO) Removing backups older than 3 days
,/config/backups/worlds-20210305-040220.zip
,03/08/2021 00:02:40: RPC_Disconnect 
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:7822 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:8406 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:8404 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:8408 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:8402 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:8407 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:8405 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:8403 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:661 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:8401 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:8258 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:8400 owner 115916012
,03/08/2021 00:02:40: Destroying abandoned non persistent zdo 115916012:7823 owner 115916012
,03/08/2021 00:02:40: Disposing socket
,03/08/2021 00:02:40: Closing socket 76561198059085861
,03/08/2021 00:02:40:   send queue size:0
,03/08/2021 00:02:40: Disposing socket
,03/08/2021 00:02:40: Got status changed msg k_ESteamNetworkingConnectionState_ClosedByPeer
,03/08/2021 00:02:40: Socket closed by peer Steamworks.SteamNetConnectionStatusChangedCallback_t
,03/08/2021 00:02:40: Got status changed msg k_ESteamNetworkingConnectionState_None
,03/08/2021 00:02:51:  Connections 2 ZDOS:384361  sent:0 recv:391
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 607.077328 ms (FindLiveObjects: 67.490991 ms CreateObjectMapping: 19.333835 ms MarkObjects: 519.502996 ms  DeleteObjects: 0.747865 ms)
,03/08/2021 00:02:53: clone 607
,03/08/2021 00:02:54: Saved 384346 zdos
,03/08/2021 00:02:54: World saved ( 1359.05ms )
,03/08/2021 00:12:27: Got status changed msg k_ESteamNetworkingConnectionState_Connecting
,03/08/2021 00:12:27: New connection
,03/08/2021 00:12:27: Accepting connection k_EResultOK
,03/08/2021 00:12:27: Connecting to Steamworks.SteamNetworkingIdentity
,03/08/2021 00:12:27: Got status changed msg k_ESteamNetworkingConnectionState_Connected
,03/08/2021 00:12:27: Connected
,03/08/2021 00:12:27: Got connection SteamID 76561198059085861
,03/08/2021 00:12:28: Got handshake from client 76561198059085861
,03/08/2021 00:12:30: VERSION check their:0.147.3  mine:0.147.3
,03/08/2021 00:12:30: Server: New peer connected,sending global keys
,03/08/2021 00:12:48: Got character ZDOID from Bacon : -431591459:1
,03/08/2021 00:12:51:  Connections 3 ZDOS:384397  sent:432 recv:564
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116805.
,Total: 629.428904 ms (FindLiveObjects: 70.790901 ms CreateObjectMapping: 20.237680 ms MarkObjects: 537.677834 ms  DeleteObjects: 0.721185 ms)
,Mar  8 00:15:00 c2c76113de28 cron.info crond[22]: USER root pid 2308 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 00:15:02 [24] (DEBUG) Received signal to check for update
,2021-03-08 00:15:03 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/08/2021 00:18:08: Placed locations in zone 20,47  duration 22.742 ms
,03/08/2021 00:22:52:  Connections 3 ZDOS:384991  sent:360 recv:443
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 168 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 624.639172 ms (FindLiveObjects: 67.210287 ms CreateObjectMapping: 20.709948 ms MarkObjects: 535.582552 ms  DeleteObjects: 1.135471 ms)
,03/08/2021 00:22:54: clone 612
,03/08/2021 00:22:55: Saved 384974 zdos
,03/08/2021 00:22:55: World saved ( 1322.39ms )
,03/08/2021 00:25:02: Loading dungeon
,03/08/2021 00:25:02: Dungeon loaded 0
,03/08/2021 00:25:02: Dungeon load time 0.07 ms
,03/08/2021 00:25:02: Available rooms:16
,03/08/2021 00:25:02: To place:30
,03/08/2021 00:25:02: Connections 4
,03/08/2021 00:25:02: placed 0 doors
,03/08/2021 00:25:02: Placed 20 rooms
,03/08/2021 00:25:02: Placed locations in zone 22,47  duration 91.972 ms
,03/08/2021 00:25:11: Placed locations in zone 22,48  duration 0.122 ms
,03/08/2021 00:26:18: Loading dungeon
,03/08/2021 00:26:18: Dungeon loaded 0
,03/08/2021 00:26:18: Dungeon load time 0.046 ms
,03/08/2021 00:26:18: Available rooms:16
,03/08/2021 00:26:18: To place:30
,03/08/2021 00:26:18: Connections 4
,03/08/2021 00:26:18: placed 0 doors
,03/08/2021 00:26:18: Placed 23 rooms
,03/08/2021 00:26:18: Placed locations in zone 23,49  duration 61.639 ms
,03/08/2021 00:26:18: Placed locations in zone 23,50  duration 5.998 ms
,03/08/2021 00:26:18: Placed locations in zone 23,51  duration 2.215 ms
,03/08/2021 00:26:21: Placed locations in zone 16,55  duration 0.113 ms
,03/08/2021 00:26:22: Placed locations in zone 17,55  duration 16.377 ms
,03/08/2021 00:26:33: Placed locations in zone 15,56  duration 1.468 ms
,03/08/2021 00:26:33: Placed locations in zone 16,56  duration 1.675 ms
,03/08/2021 00:26:34: Placed locations in zone 17,56  duration 0.839 ms
,03/08/2021 00:26:35: Placed locations in zone 24,48  duration 0.213 ms
,03/08/2021 00:26:35: Placed locations in zone 24,49  duration 45.146 ms
,03/08/2021 00:26:36: Placed locations in zone 24,50  duration 5.774 ms
,03/08/2021 00:26:36: Placed locations in zone 24,51  duration 1.595 ms
,03/08/2021 00:26:48: Placed locations in zone 17,57  duration 0.318 ms
,03/08/2021 00:26:48: Placed locations in zone 20,57  duration 1.596 ms
,03/08/2021 00:27:01: Placed locations in zone 17,58  duration 7.56 ms
,03/08/2021 00:27:16: Placed locations in zone 25,59  duration 9.914 ms
,03/08/2021 00:27:26: Placed locations in zone 27,54  duration 0.484 ms
,03/08/2021 00:27:32: Placed locations in zone 21,60  duration 0.285 ms
,03/08/2021 00:27:32: Placed locations in zone 23,60  duration 4.069 ms
,03/08/2021 00:27:50: Placed locations in zone 21,61  duration 2.823 ms
,03/08/2021 00:27:50: Placed locations in zone 23,61  duration 0.219 ms
,03/08/2021 00:27:50: Placed locations in zone 24,61  duration 0.974 ms
,03/08/2021 00:28:50: Placed locations in zone 35,54  duration 38.051 ms
,Mar  8 00:30:00 c2c76113de28 cron.info crond[22]: USER root pid 2748 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 00:30:03 [24] (DEBUG) Received signal to check for update
,2021-03-08 00:30:05 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/08/2021 00:30:27: Placed locations in zone 42,67  duration 16.019 ms
,03/08/2021 00:30:33: Placed locations in zone 41,70  duration 3.675 ms
,03/08/2021 00:30:40: Placed locations in zone 43,67  duration 0.604 ms
,03/08/2021 00:30:41: Placed locations in zone 43,68  duration 3.399 ms
,03/08/2021 00:30:47: Force generaeting hmap (2624.0, 0.0, 4544.0)
,03/08/2021 00:30:47: Placed locations in zone 42,71  duration 7.127 ms
,03/08/2021 00:31:06: Placed locations in zone 44,67  duration 2.379 ms
,03/08/2021 00:31:07: Placed locations in zone 44,70  duration 31.732 ms
,03/08/2021 00:31:48: Placed locations in zone 45,69  duration 0.839 ms
,03/08/2021 00:31:48: Placed locations in zone 45,70  duration 1.546 ms
,03/08/2021 00:31:48: Placed locations in zone 45,71  duration 1.1 ms
,03/08/2021 00:31:48: Placed locations in zone 45,72  duration 2.15 ms
,03/08/2021 00:31:57: Placed locations in zone 40,77  duration 2.064 ms
,03/08/2021 00:31:57: Placed locations in zone 45,77  duration 2.966 ms
,03/08/2021 00:32:06: Placed locations in zone 46,70  duration 0.868 ms
,03/08/2021 00:32:07: Placed locations in zone 46,73  duration 14.828 ms
,03/08/2021 00:32:07: Loading dungeon
,03/08/2021 00:32:07: Dungeon loaded 0
,03/08/2021 00:32:07: Dungeon load time 0.043 ms
,03/08/2021 00:32:07: Available rooms:18
,03/08/2021 00:32:07: To place:40
,03/08/2021 00:32:07: Connections 4
,03/08/2021 00:32:07: placed 4 doors
,03/08/2021 00:32:07: Placed 34 rooms
,03/08/2021 00:32:07: Placed locations in zone 46,76  duration 54.777 ms
,03/08/2021 00:32:13: Placed locations in zone 42,78  duration 24.436 ms
,03/08/2021 00:32:13: Placed locations in zone 43,78  duration 1.002 ms
,03/08/2021 00:32:13: Placed locations in zone 46,78  duration 8.153 ms
,03/08/2021 00:32:25: Placed locations in zone 47,74  duration 0.264 ms
,03/08/2021 00:32:39: Placed locations in zone 48,73  duration 2.28 ms
,03/08/2021 00:32:39: Placed locations in zone 48,75  duration 0.765 ms
,03/08/2021 00:32:40: Placed locations in zone 48,77  duration 0.741 ms
,03/08/2021 00:32:52:  Connections 3 ZDOS:409891  sent:218 recv:537
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 1530 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 644.462649 ms (FindLiveObjects: 73.146846 ms CreateObjectMapping: 20.117328 ms MarkObjects: 545.202372 ms  DeleteObjects: 5.995019 ms)
,03/08/2021 00:33:01: Placed locations in zone 49,71  duration 1.014 ms
,03/08/2021 00:33:01: Placed locations in zone 49,72  duration 29.317 ms
,03/08/2021 00:33:01: Placed locations in zone 49,73  duration 15.04 ms
,03/08/2021 00:33:13: Placed locations in zone 50,77  duration 0.124 ms
,03/08/2021 00:33:35: Placed locations in zone 51,76  duration 0.335 ms
,03/08/2021 00:33:37: Placed locations in zone 49,69  duration 1.526 ms
,03/08/2021 00:33:37: Placed locations in zone 50,69  duration 3.881 ms
,03/08/2021 00:33:40: Possible events:2
,03/08/2021 00:33:40: Random event set:army_bonemass
,03/08/2021 00:34:15: Placed locations in zone 52,69  duration 2.264 ms
,03/08/2021 00:34:15: Placed locations in zone 52,72  duration 36.786 ms
,03/08/2021 00:34:15: Placed locations in zone 52,74  duration 19.903 ms
,03/08/2021 00:34:15: Placed locations in zone 52,75  duration 0.117 ms
,03/08/2021 00:34:16: Placed locations in zone 52,76  duration 40.423 ms
,03/08/2021 00:34:16: Placed locations in zone 52,77  duration 0.371 ms
,03/08/2021 00:36:32: Placed locations in zone 53,72  duration 0.324 ms
,03/08/2021 00:36:32: Placed locations in zone 53,74  duration 112.087 ms
,03/08/2021 00:36:46: Placed locations in zone 54,72  duration 0.116 ms
,03/08/2021 00:36:46: Placed locations in zone 54,73  duration 6.906 ms
,03/08/2021 00:36:46: Placed locations in zone 54,75  duration 0.376 ms
,03/08/2021 00:37:14: Placed locations in zone 49,79  duration 1.168 ms
,03/08/2021 00:37:42: Placed locations in zone 53,68  duration 16.605 ms
,03/08/2021 00:38:04: Placed locations in zone 55,67  duration 36.723 ms
,03/08/2021 00:38:04: Placed locations in zone 55,71  duration 41.768 ms
,03/08/2021 00:38:04: Placed locations in zone 55,72  duration 38.193 ms
,03/08/2021 00:38:05: Placed locations in zone 55,74  duration 17.988 ms
,03/08/2021 00:38:28: Placed locations in zone 47,65  duration 15.644 ms
,03/08/2021 00:38:28: Placed locations in zone 50,65  duration 27.235 ms
,03/08/2021 00:38:32: Loading dungeon
,03/08/2021 00:38:32: Dungeon loaded 0
,03/08/2021 00:38:32: Dungeon load time 0.044 ms
,03/08/2021 00:38:32: Available rooms:18
,03/08/2021 00:38:32: To place:40
,03/08/2021 00:38:32: Connections 4
,03/08/2021 00:38:32: placed 9 doors
,03/08/2021 00:38:32: Placed 31 rooms
,03/08/2021 00:38:32: Placed locations in zone 56,65  duration 39.623 ms
,03/08/2021 00:38:32: Placed locations in zone 56,66  duration 1.038 ms
,03/08/2021 00:38:32: Placed locations in zone 56,67  duration 3.377 ms
,03/08/2021 00:38:32: Placed locations in zone 56,68  duration 3.139 ms
,03/08/2021 00:38:32: Placed locations in zone 56,70  duration 0.348 ms
,03/08/2021 00:38:32: Placed locations in zone 56,71  duration 0.107 ms
,03/08/2021 00:38:54: Placed locations in zone 57,64  duration 1.266 ms
,03/08/2021 00:38:58: Placed locations in zone 58,68  duration 3.37 ms
,03/08/2021 00:38:58: Loading dungeon
,03/08/2021 00:38:58: Dungeon loaded 0
,03/08/2021 00:38:58: Dungeon load time 0.054 ms
,03/08/2021 00:38:58: Available rooms:18
,03/08/2021 00:38:58: To place:40
,03/08/2021 00:38:58: Connections 4
,03/08/2021 00:38:58: placed 7 doors
,03/08/2021 00:38:58: Placed 27 rooms
,03/08/2021 00:38:58: Placed locations in zone 58,69  duration 31.881 ms
,03/08/2021 00:38:58: Placed locations in zone 58,70  duration 1.272 ms
,03/08/2021 00:39:10: Placed locations in zone 50,63  duration 2.575 ms
,03/08/2021 00:39:10: Placed locations in zone 52,63  duration 2.559 ms
,03/08/2021 00:39:11: Loading dungeon
,03/08/2021 00:39:11: Dungeon loaded 0
,03/08/2021 00:39:11: Dungeon load time 0.065 ms
,03/08/2021 00:39:11: Available rooms:16
,03/08/2021 00:39:11: To place:30
,03/08/2021 00:39:11: Connections 4
,03/08/2021 00:39:11: placed 0 doors
,03/08/2021 00:39:11: Placed 29 rooms
,03/08/2021 00:39:11: Placed locations in zone 53,63  duration 60.477 ms
,03/08/2021 00:39:11: Placed locations in zone 55,63  duration 0.765 ms
,03/08/2021 00:39:11: Placed locations in zone 56,63  duration 2.047 ms
,03/08/2021 00:39:11: Placed locations in zone 58,63  duration 1.928 ms
,03/08/2021 00:39:12: Placed locations in zone 59,63  duration 1.356 ms
,03/08/2021 00:39:12: Loading dungeon
,03/08/2021 00:39:12: Dungeon loaded 0
,03/08/2021 00:39:12: Dungeon load time 0.173 ms
,03/08/2021 00:39:12: Available rooms:18
,03/08/2021 00:39:12: To place:40
,03/08/2021 00:39:12: Connections 4
,03/08/2021 00:39:12: placed 8 doors
,03/08/2021 00:39:12: Placed 50 rooms
,03/08/2021 00:39:12: Placed locations in zone 59,64  duration 83.262 ms
,03/08/2021 00:39:13: Placed locations in zone 59,67  duration 2.487 ms
,03/08/2021 00:39:24: Loading dungeon
,03/08/2021 00:39:24: Dungeon loaded 0
,03/08/2021 00:39:24: Dungeon load time 0.044 ms
,03/08/2021 00:39:24: Available rooms:18
,03/08/2021 00:39:24: To place:40
,03/08/2021 00:39:24: Connections 4
,03/08/2021 00:39:24: placed 6 doors
,03/08/2021 00:39:24: Placed 22 rooms
,03/08/2021 00:39:24: Placed locations in zone 59,72  duration 37.308 ms
,03/08/2021 00:39:31: Placed locations in zone 60,67  duration 43.769 ms
,03/08/2021 00:39:43: Placed locations in zone 61,68  duration 0.111 ms
,03/08/2021 00:39:43: Placed locations in zone 61,71  duration 1.32 ms
,03/08/2021 00:39:54: Loading dungeon
,03/08/2021 00:39:54: Dungeon loaded 0
,03/08/2021 00:39:54: Dungeon load time 0.048 ms
,03/08/2021 00:39:54: Available rooms:16
,03/08/2021 00:39:54: To place:30
,03/08/2021 00:39:54: Connections 4
,03/08/2021 00:39:54: placed 0 doors
,03/08/2021 00:39:54: Placed 24 rooms
,03/08/2021 00:39:54: Placed locations in zone 55,62  duration 71.925 ms
,03/08/2021 00:39:54: Loading dungeon
,03/08/2021 00:39:54: Dungeon loaded 0
,03/08/2021 00:39:54: Dungeon load time 0.046 ms
,03/08/2021 00:39:54: Available rooms:16
,03/08/2021 00:39:54: To place:30
,03/08/2021 00:39:54: Connections 4
,03/08/2021 00:39:54: placed 1 doors
,03/08/2021 00:39:54: Placed 19 rooms
,03/08/2021 00:39:54: Placed locations in zone 56,62  duration 53.026 ms
,03/08/2021 00:39:55: Placed locations in zone 59,62  duration 1.178 ms
,03/08/2021 00:39:55: Placed locations in zone 60,62  duration 1.256 ms
,03/08/2021 00:41:19: Placed locations in zone 56,61  duration 2.425 ms
,03/08/2021 00:41:19: Placed locations in zone 57,61  duration 20.536 ms
,03/08/2021 00:41:19: Loading dungeon
,03/08/2021 00:41:19: Dungeon loaded 0
,03/08/2021 00:41:19: Dungeon load time 0.045 ms
,03/08/2021 00:41:19: Available rooms:16
,03/08/2021 00:41:19: To place:30
,03/08/2021 00:41:19: Connections 4
,03/08/2021 00:41:19: placed 0 doors
,03/08/2021 00:41:19: Placed 21 rooms
,03/08/2021 00:41:19: Placed locations in zone 58,61  duration 56.728 ms
,03/08/2021 00:41:19: Loading dungeon
,03/08/2021 00:41:19: Dungeon loaded 0
,03/08/2021 00:41:19: Dungeon load time 0.047 ms
,03/08/2021 00:41:19: Available rooms:16
,03/08/2021 00:41:19: To place:30
,03/08/2021 00:41:19: Connections 4
,03/08/2021 00:41:19: placed 0 doors
,03/08/2021 00:41:19: Placed 25 rooms
,03/08/2021 00:41:19: Placed locations in zone 59,61  duration 50.119 ms
,03/08/2021 00:42:53:  Connections 3 ZDOS:436098  sent:705 recv:715
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 690 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 642.908623 ms (FindLiveObjects: 77.581324 ms CreateObjectMapping: 20.476914 ms MarkObjects: 541.995280 ms  DeleteObjects: 2.854252 ms)
,03/08/2021 00:42:55: clone 694
,03/08/2021 00:42:56: Saved 436082 zdos
,03/08/2021 00:42:57: World saved ( 1735.31ms )
,Mar  8 00:45:00 c2c76113de28 cron.info crond[22]: USER root pid 3186 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 00:45:05 [24] (DEBUG) Received signal to check for update
,2021-03-08 00:45:06 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/08/2021 00:46:13: Placed locations in zone 44,79  duration 15.286 ms
,03/08/2021 00:47:32: Placed locations in zone 52,80  duration 1.669 ms
,03/08/2021 00:47:32: Placed locations in zone 50,81  duration 1.27 ms
,03/08/2021 00:48:35: Got character ZDOID from Hofflander : 0:0
,03/08/2021 00:48:43: Got character ZDOID from Hofflander : 1998091234:17305
,03/08/2021 00:52:13: Placed locations in zone 54,79  duration 0.109 ms
,03/08/2021 00:52:13: Placed locations in zone 54,80  duration 0.107 ms
,03/08/2021 00:52:40: Placed locations in zone 55,80  duration 0.263 ms
,03/08/2021 00:52:53:  Connections 3 ZDOS:438640  sent:4 recv:666
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 177 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 647.974734 ms (FindLiveObjects: 71.969435 ms CreateObjectMapping: 20.929454 ms MarkObjects: 553.917891 ms  DeleteObjects: 1.157129 ms)
,03/08/2021 00:52:56: Placed locations in zone 56,75  duration 0.118 ms
,03/08/2021 00:52:56: Placed locations in zone 56,76  duration 0.388 ms
,03/08/2021 00:52:56: Placed locations in zone 56,79  duration 40.9 ms
,03/08/2021 00:52:56: Placed locations in zone 56,80  duration 5.986 ms
,03/08/2021 00:53:12: Placed locations in zone 57,79  duration 1.247 ms
,03/08/2021 00:53:12: Placed locations in zone 57,81  duration 1.805 ms
,03/08/2021 00:53:32: Got character ZDOID from Bacon : 0:0
,03/08/2021 00:53:42: Got character ZDOID from Bacon : -431591459:3652
,03/08/2021 00:59:27: Got character ZDOID from Bacon : 0:0
,03/08/2021 00:59:29: Got character ZDOID from Bacon : -431591459:4129
,Mar  8 01:00:00 c2c76113de28 cron.info crond[22]: USER root pid 3625 cmd [ -f "/var/run/valheim-backup.pid" ] && kill -HUP $(cat /var/run/valheim-backup.pid)
,Mar  8 01:00:00 c2c76113de28 cron.info crond[22]: USER root pid 3626 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 01:00:05 [58] (DEBUG) Received signal to backup world
,2021-03-08 01:00:05 [58] (INFO) Backing up Valheim server worlds to /config/backups/worlds-20210308-010005.zip
,  adding: worlds/ (stored 0%)
,  adding: worlds/Dedicated.fwl.old (deflated 4%)
,  adding: worlds/Dedicated.fwl (deflated 4%)
,  adding: worlds/Dedicated.db.old2021-03-08 01:00:07 [24] (DEBUG) Received signal to check for update
, (deflated 69%)
,  adding: worlds/Dedicated.db2021-03-08 01:00:08 [24] (DEBUG) Players connected to Valheim server - skipping update check
, (deflated 69%)
,2021-03-08 01:00:08 [58] (INFO) Removing backups older than 3 days
,/config/backups/worlds-20210305-050007.zip
,/config/backups/worlds-20210305-060003.zip
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 50ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 50ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 50ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 50ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 51ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 51ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 52ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 52ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 52ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 52ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 53ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 53ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 53ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 53ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 54ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 54ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 54ms.  This directly adds to delay of processing of network packets!
,src/steamnetworkingsockets/clientlib/steamnetworkingsockets_lowlevel.cpp (1276) : Assertion Failed: SDR service thread gave up on lock after waiting 54ms.  This directly adds to delay of processing of network packets!
,03/08/2021 01:01:31: RPC_Disconnect 
,03/08/2021 01:01:31: Destroying abandoned non persistent zdo -431591459:4227 owner -431591459
,03/08/2021 01:01:31: Destroying abandoned non persistent zdo -431591459:4247 owner -431591459
,03/08/2021 01:01:31: Destroying abandoned non persistent zdo -431591459:4296 owner -431591459
,03/08/2021 01:01:31: Destroying abandoned non persistent zdo -431591459:4129 owner -431591459
,03/08/2021 01:01:31: Destroying abandoned non persistent zdo -431591459:4297 owner -431591459
,03/08/2021 01:01:31: Disposing socket
,03/08/2021 01:01:31: Closing socket 76561198059085861
,03/08/2021 01:01:31:   send queue size:0
,03/08/2021 01:01:31: Disposing socket
,03/08/2021 01:01:31: Got status changed msg k_ESteamNetworkingConnectionState_ClosedByPeer
,03/08/2021 01:01:31: Socket closed by peer Steamworks.SteamNetConnectionStatusChangedCallback_t
,03/08/2021 01:01:31: Got status changed msg k_ESteamNetworkingConnectionState_None
,03/08/2021 01:02:54:  Connections 2 ZDOS:439856  sent:32 recv:35
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 57 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 647.027162 ms (FindLiveObjects: 76.313733 ms CreateObjectMapping: 20.668793 ms MarkObjects: 549.229174 ms  DeleteObjects: 0.814588 ms)
,03/08/2021 01:02:56: clone 747
,03/08/2021 01:02:58: Saved 439852 zdos
,03/08/2021 01:02:58: World saved ( 1693.031ms )
,03/08/2021 01:07:48: Time 296778.999860771, day:164    nextm:297270.000010729  skipspeed:40.916679163153
,03/08/2021 01:11:31: Got status changed msg k_ESteamNetworkingConnectionState_Connecting
,03/08/2021 01:11:31: New connection
,03/08/2021 01:11:31: Accepting connection k_EResultOK
,03/08/2021 01:11:31: Connecting to Steamworks.SteamNetworkingIdentity
,03/08/2021 01:11:31: Got status changed msg k_ESteamNetworkingConnectionState_Connected
,03/08/2021 01:11:31: Connected
,03/08/2021 01:11:31: Got connection SteamID 76561198059085861
,03/08/2021 01:11:31: Got handshake from client 76561198059085861
,03/08/2021 01:11:34: VERSION check their:0.147.3  mine:0.147.3
,03/08/2021 01:11:34: Server: New peer connected,sending global keys
,03/08/2021 01:11:52: Got character ZDOID from Bacon : 193342549:1
,03/08/2021 01:12:54:  Connections 3 ZDOS:439970  sent:333 recv:667
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 631.159533 ms (FindLiveObjects: 67.166983 ms CreateObjectMapping: 19.207088 ms MarkObjects: 544.087667 ms  DeleteObjects: 0.696827 ms)
,Mar  8 01:15:00 c2c76113de28 cron.info crond[22]: USER root pid 4079 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 01:15:08 [24] (DEBUG) Received signal to check for update
,2021-03-08 01:15:10 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/08/2021 01:15:36: Placed locations in zone 58,78  duration 0.125 ms
,03/08/2021 01:22:55:  Connections 3 ZDOS:440439  sent:406 recv:338
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 57 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 644.393594 ms (FindLiveObjects: 71.856795 ms CreateObjectMapping: 20.502505 ms MarkObjects: 551.148701 ms  DeleteObjects: 0.884699 ms)
,03/08/2021 01:22:58: clone 826
,03/08/2021 01:22:59: Saved 440391 zdos
,03/08/2021 01:22:59: World saved ( 1583.103ms )
,Mar  8 01:30:00 c2c76113de28 cron.info crond[22]: USER root pid 4515 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 01:30:01 [24] (DEBUG) Received signal to check for update
,2021-03-08 01:30:02 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/08/2021 01:31:54: RPC_Disconnect 
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2489 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2479 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2476 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2485 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:1 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2499 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2478 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2208 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2470 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2483 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2492 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2493 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2488 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2491 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2500 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2484 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2490 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2498 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2497 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2495 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2477 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2481 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2486 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2472 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2496 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2480 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2487 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2494 owner 193342549
,03/08/2021 01:31:54: Destroying abandoned non persistent zdo 193342549:2482 owner 193342549
,03/08/2021 01:31:54: Disposing socket
,03/08/2021 01:31:54: Closing socket 76561198059085861
,03/08/2021 01:31:54:   send queue size:0
,03/08/2021 01:31:54: Disposing socket
,03/08/2021 01:31:54: Got status changed msg k_ESteamNetworkingConnectionState_ClosedByPeer
,03/08/2021 01:31:54: Socket closed by peer Steamworks.SteamNetConnectionStatusChangedCallback_t
,03/08/2021 01:31:54: Got status changed msg k_ESteamNetworkingConnectionState_None
,03/08/2021 01:32:56:  Connections 2 ZDOS:440505  sent:2 recv:194
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 33 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 656.795582 ms (FindLiveObjects: 73.010601 ms CreateObjectMapping: 20.780743 ms MarkObjects: 562.234844 ms  DeleteObjects: 0.768528 ms)
,03/08/2021 01:42:56:  Connections 2 ZDOS:440559  sent:775 recv:356
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 652.290318 ms (FindLiveObjects: 75.675905 ms CreateObjectMapping: 20.486714 ms MarkObjects: 555.361458 ms  DeleteObjects: 0.765217 ms)
,03/08/2021 01:42:59: clone 700
,03/08/2021 01:43:00: Saved 440553 zdos
,03/08/2021 01:43:01: World saved ( 1763.087ms )
,Mar  8 01:45:00 c2c76113de28 cron.info crond[22]: USER root pid 4955 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 01:45:02 [24] (DEBUG) Received signal to check for update
,2021-03-08 01:45:04 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/08/2021 01:52:57:  Connections 2 ZDOS:440543  sent:1 recv:303
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 659.529640 ms (FindLiveObjects: 77.666319 ms CreateObjectMapping: 20.607653 ms MarkObjects: 560.582481 ms  DeleteObjects: 0.671890 ms)
,Mar  8 02:00:00 c2c76113de28 cron.info crond[22]: USER root pid 5392 cmd [ -f "/var/run/valheim-backup.pid" ] && kill -HUP $(cat /var/run/valheim-backup.pid)
,Mar  8 02:00:00 c2c76113de28 cron.info crond[22]: USER root pid 5393 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 02:00:02 [58] (DEBUG) Received signal to backup world
,2021-03-08 02:00:02 [58] (INFO) Backing up Valheim server worlds to /config/backups/worlds-20210308-020002.zip
,  adding: worlds/ (stored 0%)
,  adding: worlds/Dedicated.fwl.old (deflated 4%)
,  adding: worlds/Dedicated.fwl (deflated 4%)
,  adding: worlds/Dedicated.db.old (deflated 69%)
,  adding: worlds/Dedicated.db2021-03-08 02:00:04 [24] (DEBUG) Received signal to check for update
, (deflated 69%)
,2021-03-08 02:00:05 [58] (INFO) Removing backups older than 3 days
,2021-03-08 02:00:05 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/08/2021 02:02:57:  Connections 2 ZDOS:440576  sent:0 recv:435
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 644.263030 ms (FindLiveObjects: 73.364836 ms CreateObjectMapping: 19.362278 ms MarkObjects: 550.812321 ms  DeleteObjects: 0.722441 ms)
,03/08/2021 02:03:00: clone 729
,03/08/2021 02:03:02: Saved 440546 zdos
,03/08/2021 02:03:02: World saved ( 1734.042ms )
,03/08/2021 02:12:58:  Connections 2 ZDOS:440563  sent:114 recv:132
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 638.355514 ms (FindLiveObjects: 75.246393 ms CreateObjectMapping: 19.887768 ms MarkObjects: 542.546982 ms  DeleteObjects: 0.673568 ms)
,Mar  8 02:15:00 c2c76113de28 cron.info crond[22]: USER root pid 5844 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 02:15:06 [24] (DEBUG) Received signal to check for update
,2021-03-08 02:15:07 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/08/2021 02:22:58:  Connections 2 ZDOS:440603  sent:613 recv:252
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 637.121844 ms (FindLiveObjects: 74.313370 ms CreateObjectMapping: 20.015046 ms MarkObjects: 542.092642 ms  DeleteObjects: 0.699864 ms)
,03/08/2021 02:23:01: clone 739
,03/08/2021 02:23:03: Saved 440587 zdos
,03/08/2021 02:23:03: World saved ( 1795.588ms )
,Mar  8 02:30:00 c2c76113de28 cron.info crond[22]: USER root pid 6284 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 02:30:07 [24] (DEBUG) Received signal to check for update
,2021-03-08 02:30:08 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/08/2021 02:32:59:  Connections 2 ZDOS:440627  sent:1 recv:220
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 647.775800 ms (FindLiveObjects: 70.782043 ms CreateObjectMapping: 20.927195 ms MarkObjects: 555.452263 ms  DeleteObjects: 0.613399 ms)
,03/08/2021 02:42:59:  Connections 2 ZDOS:440597  sent:29 recv:332
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 643.676777 ms (FindLiveObjects: 76.273259 ms CreateObjectMapping: 20.331189 ms MarkObjects: 546.297715 ms  DeleteObjects: 0.773382 ms)
,03/08/2021 02:43:03: clone 697
,03/08/2021 02:43:04: Saved 440580 zdos
,03/08/2021 02:43:04: World saved ( 1740.136ms )
,Mar  8 02:45:00 c2c76113de28 cron.info crond[22]: USER root pid 6722 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 02:45:09 [24] (DEBUG) Received signal to check for update
,2021-03-08 02:45:10 [24] (DEBUG) Players connected to Valheim server - skipping update check
,03/08/2021 02:51:41: Possible events:2
,03/08/2021 02:51:41: Random event set:army_bonemass
,03/08/2021 02:53:00:  Connections 2 ZDOS:440668  sent:190 recv:308
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 656.280256 ms (FindLiveObjects: 78.702814 ms CreateObjectMapping: 21.355673 ms MarkObjects: 555.498048 ms  DeleteObjects: 0.722454 ms)
,03/08/2021 02:54:30: Got character ZDOID from Hofflander : 0:0
,03/08/2021 02:54:39: Got character ZDOID from Hofflander : 1998091234:31483
,Mar  8 03:00:00 c2c76113de28 cron.info crond[22]: USER root pid 7159 cmd [ -f "/var/run/valheim-backup.pid" ] && kill -HUP $(cat /var/run/valheim-backup.pid)
,Mar  8 03:00:00 c2c76113de28 cron.info crond[22]: USER root pid 7160 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 03:00:01 [24] (DEBUG) Received signal to check for update
,2021-03-08 03:00:03 [24] (DEBUG) Players connected to Valheim server - skipping update check
,2021-03-08 03:00:06 [58] (DEBUG) Received signal to backup world
,2021-03-08 03:00:06 [58] (INFO) Backing up Valheim server worlds to /config/backups/worlds-20210308-030006.zip
,  adding: worlds/ (stored 0%)
,  adding: worlds/Dedicated.fwl.old (deflated 4%)
,  adding: worlds/Dedicated.fwl (deflated 4%)
,  adding: worlds/Dedicated.db.old (deflated 69%)
,  adding: worlds/Dedicated.db (deflated 69%)
,2021-03-08 03:00:10 [58] (INFO) Removing backups older than 3 days
,/config/backups/worlds-20210305-070007.zip
,/config/backups/worlds-20210305-080002.zip
,03/08/2021 03:03:00:  Connections 2 ZDOS:440655  sent:416 recv:391
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 656.172847 ms (FindLiveObjects: 71.445936 ms CreateObjectMapping: 19.939931 ms MarkObjects: 564.114107 ms  DeleteObjects: 0.671709 ms)
,03/08/2021 03:03:04: clone 695
,03/08/2021 03:03:05: Saved 440645 zdos
,03/08/2021 03:03:06: World saved ( 1739.626ms )
,03/08/2021 03:07:30: RPC_Disconnect 
,03/08/2021 03:07:30: Destroying abandoned non persistent zdo 1998091234:32277 owner 1998091234
,03/08/2021 03:07:30: Destroying abandoned non persistent zdo 1998091234:32279 owner 1998091234
,03/08/2021 03:07:31: Destroying abandoned non persistent zdo 1998091234:32276 owner 1998091234
,03/08/2021 03:07:31: Destroying abandoned non persistent zdo 1998091234:31483 owner 1998091234
,03/08/2021 03:07:31: Destroying abandoned non persistent zdo 1998091234:32278 owner 1998091234
,03/08/2021 03:07:31: Disposing socket
,03/08/2021 03:07:31: Closing socket 76561197963260490
,03/08/2021 03:07:31:   send queue size:0
,03/08/2021 03:07:31: Disposing socket
,03/08/2021 03:07:31: Got status changed msg k_ESteamNetworkingConnectionState_ClosedByPeer
,03/08/2021 03:07:31: Socket closed by peer Steamworks.SteamNetConnectionStatusChangedCallback_t
,03/08/2021 03:07:31: Got status changed msg k_ESteamNetworkingConnectionState_None
,03/08/2021 03:12:30: RPC_Disconnect 
,03/08/2021 03:12:30: Destroying abandoned non persistent zdo 121322511:1 owner 121322511
,03/08/2021 03:12:30: Disposing socket
,03/08/2021 03:12:30: Closing socket 76561197993893753
,03/08/2021 03:12:30:   send queue size:0
,03/08/2021 03:12:30: Disposing socket
,03/08/2021 03:12:30: Got status changed msg k_ESteamNetworkingConnectionState_ClosedByPeer
,03/08/2021 03:12:30: Socket closed by peer Steamworks.SteamNetConnectionStatusChangedCallback_t
,03/08/2021 03:12:30: Got status changed msg k_ESteamNetworkingConnectionState_None
,03/08/2021 03:13:01:  Connections 0 ZDOS:440656  sent:0 recv:0
,Unloading 0 Unused Serialized files (Serialized files now loaded: 0)
,Unloading 0 unused Assets to reduce memory usage. Loaded Objects now: 1116798.
,Total: 657.212835 ms (FindLiveObjects: 73.129925 ms CreateObjectMapping: 20.091044 ms MarkObjects: 563.293140 ms  DeleteObjects: 0.697364 ms)
,Mar  8 03:15:00 c2c76113de28 cron.info crond[22]: USER root pid 7615 cmd [ -f "/var/run/valheim-updater.pid" ] && kill -HUP $(cat /var/run/valheim-updater.pid)
,2021-03-08 03:15:03 [24] (DEBUG) Received signal to check for update
,2021-03-08 03:15:06 [24] (DEBUG) No players connected to Valheim server
,2021-03-08 03:15:06 [24] (INFO) Downloading/updating/validating Valheim server from Steam
,2021-03-08 03:15:06 [24] (DEBUG) Found CPU with 3447.461 MHz
,Redirecting stderr to '/root/Steam/logs/stderr.txt'
,[  0%] Checking for available updates...
,[----] Verifying installation...
,Steam Console Client (c) Valve Corporation
,-- type 'quit' to exit --
,Loading Steam API...OK.
,
,Connecting anonymously to Steam Public...Logged in OK
,Waiting for user info...OK
, Update state (0x5) verifying install, progress: 0.20 (2097152 / 1051599543)
, Update state (0x5) verifying install, progress: 16.77 (176346948 / 1051599543)
, Update state (0x5) verifying install, progress: 42.59 (447881783 / 1051599543)
, Update state (0x5) verifying install, progress: 69.78 (733786495 / 1051599543)
, Update state (0x5) verifying install, progress: 97.47 (1024959135 / 1051599543)
,Success! App '896660' fully installed.
,.d..t...... ./
,2021-03-08 03:15:19 [24] (INFO) Valheim Server is already the latest version
,

**Comment by lloesche** [comment]
> I can provide the logs for the current run, but I don't know what good that will do.

unfortunately nothing really. We would need the logs that tell us why the server quit unexpectedly. How much memory does your container have? If you pull the latest container image we write memory statistics on every update check. Maybe those can give us an idea of what causes your server to crash.
I went over the entire log you posted and other than `Warning: failed to set thread priority` I don't see anything wrong with it. This warning can be avoided by giving the server `CAP_SYS_NICE` (`--cap-add=sys_nice`). This will allow the Steam networking library to give the thread that processes network packets higher CPU priority. But we've been running our server without it for weeks and not had any issues. So I doubt it's got anything to do with your crashes.

Also to avoid confusion, the issue @altrae was having above is not related to this one.
He just didn't know there was a daily cron that restarts the server at 5 AM each morning. We do those restarts to soften the effects of potential resource leaks in Valheim server.

**Comment by lloesche** [comment]
After #84 I'm relatively convinced that all "Server restarts intermittently" and "Server disconnected players intermittently" errors fall into the "not enough resources" category.
If the server has two or more 2 GHz+ cores and 8 GB RAM dedicated to it and still restarts intermittently lets open a new issue.

**Comment by EnigmaRaptor** [comment]
For clarification, The system I am running on has an i5-5600 and 16GB of RAM.  I did not set a dedicated RAM for valheim, so it's possible that it could make a difference to do so.  However, setting the SERVER_PUBLIC var to 1 seems to have fixed it.  I also changed UPDATE_IF_IDLE to true, so I can't rule that out either.

**Comment by coatmaker618** [comment]
Wanted to mention that I was struggling with a slightly different networking issue for a few days, but it 100% ended up being a resource constraint.

I had trouble finding this post, and was finding solutions like "roll back Valheim" which isn't practical :(

I ask this genuinely as I'm a fake dev--but is there any way to notify when resources are full? It is very not obvious from what I was seeing (but I sure won't forget).

**Comment by Kiaos** [comment]
Mine restarts at 10:00 PM mountain time each night
